source/index.html

```diff
@@ -1,12 +1,230 @@
 <!DOCTYPE html>
-<html lang="ru">
+<html class="page" lang="ru">
   <head>
     <meta charset="utf-8">
     <title>HTML Academy: Седона</title>
+    <link href="https://fonts.googleapis.com/css?family=PT+Sans:400,700&amp;subset=latin,cyrillic" rel="stylesheet">
+    <link rel="stylesheet" href="css/style.css">
   </head>
   <body>
-
-    <p>Репозиторий создан для обучения на профессиональном онлайн‑курсе «<a href="https://htmlacademy.ru/intensive/adaptive">HTML и CSS. Адаптивная вёрстка и автоматизация</a>».</p>
-
+    <div class="page__body-container">
+      <header class="page-header">
+        <a class="page-header__logo">
+          <img src="./img/logotype.svg" width="102" height="84" alt="логотип города Седона»">
+        </a>
+        <nav class="main-nav" aria-label="навигация по сайту">
+          <button class="main-nav__toggle" type="button">
+            <span class="visually-hidden">
+              Открыть меню
+            </span>
+          </button>
+          <div class="main-nav__wrapper">
+            <ul class="main-nav__list">
+              <li class="main-nav__item main-nav__item--mobile">
+                <a class="main-nav__link" href="#">
```

Comment:
```suggestion
                <a class="main-nav__link">
```

мы на главной, хреф не надо

Reply:
Сделал.